go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/cosmos/ibc-apps/modules/rate-limiting/v8 v8.0.0
 	github.com/cosmos/ibc-go/modules/capability v1.0.0
 	github.com/cosmos/ibc-go/v8 v8.4.0
-	github.com/cosmos/interchain-security/v5 v5.1.1
+	github.com/cosmos/interchain-security/v5 v5.0.0-20240806104629-29327696b8e6
```

Comment:
This looks like a regression, but this is just temporary and stems from upgrading to a commit on main instead of teh v5.1 branch

This will be updated when we move to the actual ICS release